src/main/io/displayport_msp.c

```diff
@@ -165,20 +165,14 @@ static bool isSynced(const displayPort_t *displayPort)
 
 static void redraw(displayPort_t *displayPort)
 {
-#ifdef USE_OSD
     if (vcdProfile()->video_system == VIDEO_SYSTEM_HD) {
         displayPort->rows = osdConfig()->canvas_rows;
```

Comment:
Try building without any OSD defined.

Reply:
Works for me.

Reply:
Then USE_OSD is being included regardless.

As: PG_REGISTER_WITH_RESET_FN(osdConfig_t, osdConfig, PG_OSD_CONFIG, 12); is within a USE_OSD define.

Reply:
Actually - no I see the protection added in common_pre.h USE_MSP_DISPLAYPORT is not defined now unless USE_OSD is.

Reply:
Is it legitimate to have CMS, but no OSD?

Reply:
Good question. I say not.